docs/integrations/sources/s3.md

```diff
@@ -345,6 +345,7 @@ This connector utilizes the open source [Unstructured](https://unstructured-io.g
 
 | Version | Date       | Pull Request                                                                                                    | Subject                                                                                                              |
 | :------ | :--------- | :-------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------- |
+| 4.6.0 | 2024-06-21 | [39573](https://github.com/airbytehq/airbyte/pull/39573) | Update to Airbyte CDK 2.0.0 |
```

Comment:
nit: the date is in the past

nit: I would add that the main goal is to improve performance

Reply:
Good callouts! Will update